FirebaseVertexAI/Sources/GenerativeAIService.swift

```diff
@@ -155,9 +161,10 @@ struct GenerativeAIService {
     var urlRequest = URLRequest(url: request.url)
     urlRequest.httpMethod = "POST"
     urlRequest.setValue(apiKey, forHTTPHeaderField: "x-goog-api-key")
-    // TODO: Determine the right client header to use.
-    //    urlRequest.setValue("genai-swift/\(GenerativeAISwift.version))",
-    //                        forHTTPHeaderField: "x-goog-api-client")
+    urlRequest.setValue(
+      "\(GenerativeAIService.languageTag) \(GenerativeAIService.firebaseVersionTag)",
```

Comment:
Here

Reply:
Changing it to `gl-swift/5` in https://github.com/firebase/firebase-ios-sdk/pull/12829. Thanks for verifying!